web-app/src/components/organisms/AuthPagesSignInForm.vue

```diff
@@ -0,0 +1,26 @@
+<script setup="ts">
+  import InputWithLabel from '../molecules/InputWithLabel.vue';
+
+  // Props & Emitters
+  defineProps(['email', 'password']);
```

Comment:
One thing I'm trying to include going forward is prop typing & validation.
You can read more here: https://vuejs.org/guide/components/props.html#prop-validation